client/src/AppRoutes.tsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react'
 import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import Notion from './Notion'
+import Substack from './Substack'
 import Manage from './Manage'
+import config from '../config'
+
+const notion = config.embedPlatform === 'notion'
 
 const AppRoutes: React.FC = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path={'/manage'} element={ <Manage />} />
-        <Route path='/*' element={ <Notion />} />
+        <Route path='/*' element={ notion ? <Notion /> : <Substack /> } />
```

Comment:
It needs to read from contract to figure out what to render, not from a fixed config

Reply:
Yes, we can.
My question is the smart contract query will take some time (of course not much) and since the frontend domain is specific to what to render, we can reduce the load time if we specify what to render directly in the env

Reply:
All SLDs (i.e. domains) point to the same frontend (client), so you can't decide what to render in the env. It must be determined from the SLD and querying smart contract. See implementation for notion, and test it yourself on some .country domains, e.g. www.h.country, web.zeroknowledge.country, www.thelibrary.country - they all point to the exactly the same frontend deployment (which we deployed, not the user)

Reply:
Okay